### DISCUSSION TITLE
Request for modification of Example in document of "asyncio"

### DISCUSSION THREAD

**Discussion by SoratobuMacaroniPenguin** [micropython/micropython#13579] [General]
I only tested the ESP8266, so it might be a problem only with the ESP8266.
I'm a beginner and I stumbled, so please correct me.

last line of the "asyncio" example.
asyncio.run(main(Pin(1), Pin(2)))
of
asyncio.run(main(Pin(1,Pin.OUT), Pin(2,Pin.OUT)))

Thank you for your support.

**Comment by jimmo** [comment]
@SoratobuMacaroniPenguin That's a good suggestion. Are you interested in submitting a PR? Otherwise we can turn this into an issue.
